JavaScript/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    env: {
+        browser: true,
+    },
+    extends: [
+        'airbnb',
+        'plugin:jsdoc/recommended',
+    ],
+    rules: {
+        'max-len': ['error', {
```

Comment:
may move formatting rules in prettier or editor config and integrate it in eslint
https://prettier.io/docs/en/integrating-with-linters.html

Reply:
how it would look with prettier?

Reply:
prettier is an additional dependency
I see no problem if we stay the basic config in eslint and in your project, you can customize it